File: Versão em JavaScript/Tarefa - Exercicios/Tarefa-Exercicio-4.js
// Crie um algoritmo que permita ao usuário
// entrar com o valor e a quantidade de um produto.

const prompt = require('prompt-sync')({ sigint: true })

const nome = prompt('Digite o nome do produto ')

const quantidade = prompt('Digite a quantidade do produto ')

const valor = prompt('Agora para finalizarmos, digite o valor do produto ')

const totalCompra = (valor * quantidade)

console.log('===========================================================')

console.log('Estamos calculando e gerando o resultado')
console.log('3')
console.log('2')
console.log('1')

console.log(`O valor de ${quantidade} unidades de ${nome} é ${totalCompra}`)

File: Versão em Python/Tarefa - Exercicio/Tarefa-Exercicio-4.py
# Crie um algoritmo que permita ao usuário
# entrar com o valor e a quantidade de um produto.

nomeProduto = input('Digite o nome do produto: ')
print('----------------------------------------')
valor = input('Digite o valor do produto: ')
print('----------------------------------------')
quantidade = input('Agora digite a quantidade do produto, para finalizarmos ')
print('----------------------------------------')

x = int(valor)

y = int(quantidade)

total = (x * y)

print('O valor de', quantidade, "unidades", nomeProduto, "é", total, "R$")
